src/main/resources/hudson/security/DangerousMatrixPermissionsAdministrativeMonitor/message.properties

```diff
@@ -1,8 +1,10 @@
-blurb = Your authorization strategy is currently configured to grant at least one of the permissions that can be used to escalate permissions and execute arbitrary code to users without Overall/Administer permission: \
+blurb = Your jenkins is currently configured to enable granting at least one of the permissions that can be used to escalate permissions and execute arbitrary code to users without Overall/Administer permission: \
   <ul> \
     <li>Run Scripts</li> \
     <li>Configure Update Center</li> \
     <li>Upload Plugins</li> \
   </ul> \
-  These permissions will no longer actually be granted to users without Overall/Administer permission for security reasons. \
-  It is recommended that you remove these permissions, or grant Overall/Administer to affected users or groups:
\ No newline at end of file
+  <br\> \
```

Comment:
I think you still need to say the above as well as that they are to be removed.  the plugin could be installed on an older version of Jenkins and then the user would not know why `bob` is no longer able to `RUN_SCRIPTS`